README.md

```diff
@@ -1,12 +1,9 @@
 itkMGHImageIO
 =============
 
+![Build, test, package status](https://github.com/Slicer/itkMGHImageIOZZ/workflows/Build,%20test,%20package/badge.svg)
```

Comment:
Look  like `itkMGHImageIOZZ` should be changed to `itkMGHImageIO` ?

Reply:
Good catch :+1: My vim usage is escaping :keyboard: :keycap_ten: